tests/integration/test_utils/data_source_config/postgres.py

```diff
@@ -132,11 +132,15 @@ def pytest_mark(self) -> pytest.MarkDecorator:
 
     @override
     def create_batch_setup(
-        self, data: pd.DataFrame, request: pytest.FixtureRequest
+        self,
+        request: pytest.FixtureRequest,
+        data: pd.DataFrame,
+        extra_data: Mapping[str, pd.DataFrame],
```

Comment:
I vaguely remember the conversation around possibly using inheritance to separate out SQL datasources - could you let me know why we decided to update the base class? Concerns about Liskov?